plugins/src/webrtcsink/imp.rs

```diff
@@ -461,7 +463,7 @@ impl VideoEncoder {
 
         let width = height.mul_div_ceil(ratio.numer(), ratio.denom()).unwrap();
 
-        width + 1 >> 1 << 1
+        (width + 1) & !1
```

Comment:
All the shifting you did was a bit confusing and clippy complained that some parenthesis should be added to make it less confusing, i.e. `((width + 1) >> 1) << 1`. But IMHO with the binary `&` it's clearer what happens here.

Reply:
works for me, though tbh it took me a second to understand what `!1` did :)